### PR TITLE
add note that ownCloud is yours and needs your testing...

### PR DESCRIPTION
**Pull request by jospoortvliet**
@LukasReschke here you go. What do you think?

I essentially argue that, as an open source project, ownCloud gives you ownership and thus responsibility when you use it. That means your fait is in your own hands - if you don't help us test your use case before release, there is no guarantee that it will work when we declare a release stable. This in response to people blaming Inc for an unstable release - Inc has only responsibility to its paying customers *

@karlitschek @jancborchardt I would appreciate your opinions too.

* and to fellow ownCloud contributors - to be a good citizen in ownCloud is of course important. Things aren't as simple as a few sentences can paint them...

### CODE REVIEW

**Comment by LukasReschke** [comment]
I like it - thanks :+1:

File: page-faq.php [hunk 1]
```diff
@@ -85,6 +86,13 @@
 	<li>The built in Update app will usually have new ownCloud releases available for upgrade some time in the week after a major release.</li>
 </ul>
 
+<a name="upgradeyet"></a>
+<h3>Is it safe to upgrade to the new release?</h3>
+<p>All code entering ownCloud has been reviewed twice as well as tested by both automated means and through the efforts of volunteers. Releases are done only when all issues we are aware off are fixed or can be mitigated easily. In short, ownCloud releases are always as stable as we could make it without <em>your</em> help. If you want the guarantee that it will work in your specific situation, <a href="<?php echo $DOCUMENTATION_DEVELOPER; ?>testing">make sure you help test before a release is made</a> so your issue can be fixed in time.</p>
```
Comment (jancborchardt) [inline page-faq.php]: »issues we are aware of« with one f instead of »off«. ;)

**Comment by jospoortvliet** [comment]
@jancborchardt if that's the worst you see there, I fixed it - you good with merging?

**Comment by jancborchardt** [comment]
Yeah, looks good! :+1:

**Pull request merged**